py3_test_reqs.txt

```diff
@@ -0,0 +1,2 @@
+six==1.10.0
+git+https://github.com/python/mypy.git@5d2dfce8cf6b9a0a7ce8fe8e3cb5866094459f8c
```

Comment:
I'm a bit doubtful about the name `py3_test_reqs.txt`. Do you think the name is good?

You also pointed out that we might have multiple requirements files in the future (separate for testing and production and maybe (hopefully not) separate for python 2 and python 3). Maybe we can have a directory 'setup/' which contains all files needed to set Zulip up, like the multiple requirements files, provision.py, zulip_tools.py, Vagrantfile, etc. This file belongs there more according to me.

But for now, should I move it to tools/ or should I let it stay where it is?